hip-service/Link/Patient/LinkPatientController.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Threading.Tasks;
+using hip_service.Link.Patient.Dto;
+using HipLibrary.Patient;
+using HipLibrary.Patient.Models.Request;
```

Comment:
Can be separate PR to fix the convention.

Reply:
Done